transport/http/transport.go

```diff
@@ -50,6 +54,16 @@ type TransportOption func(*transportConfig)
 
 func (TransportOption) httpOption() {}
 
+// NilOption is empty option - example use case:
+// opt := NilOption
```

Comment:
Sometimes I don't understand github. I replied inline but those replies don't show up in "conversation", so I'll re-add them here:

Ok, I think I understood what you wanted - updated in next patch.